src/karapace/kafka_rest_apis/consumer_manager.py

```diff
@@ -291,6 +294,13 @@ async def commit_offsets(
             payload = payload or None
             try:
                 await consumer.commit(offsets=payload)
+            except UnknownError as ue:
+                error_trace = traceback.format_exc()  # Capture the full stack trace
+                offset_stored_err_message = "Commit failed: Local: No offset stored"
+                if offset_stored_err_message in error_trace:
+                    LOG.warning("Ignoring KafkaError: No offset stored. %s", internal_name)
```

Comment:
It'd be nice to add a unit test for this log line.

Reply:
Indeed. Updated with caplog.